apps/gro/steps.js

```diff
@@ -152,7 +152,59 @@ module.exports = {
   '/confirm': {
     controller: require('./controllers/confirm'),
     template: 'confirm.html',
-    next: '/confirmation'
+    next: '/confirmation',
+    modelData: {
```

Comment:
Is `modelData` the right name? Perhaps just `data` or `meta`? I know it's only a name, but this essentially config, and calling it `modelData` feels like it blurs the line too much. What do you think?